src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,139 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
```

Comment:
why do we need to setLoading on `componentDidMount` loading `true`. As for me, better to set true as initial value when component mounting, to avoid small jumps of the content
```suggestion
  const [loading, setLoading] = useState(true);
  const [people, setPeople] = useState<Person[]>([]);
  const [error, setError] = useState(false);

  useEffect(() => {
```